### ISSUE TITLE
Ignore dependabot patch versions?

### ISSUE THREAD

**Issue by pdehaan** [mozilla/fxa#13897]
## Description

I think our current [config](https://github.com/mozilla/fxa/blob/45cbc1c5a2e716b54b467295835f161a578d9c08/.github/dependabot.yml#L7) limits us to 6 open ["dependency"](https://github.com/mozilla/fxa/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Adependencies) PRs at a single time.

https://github.com/mozilla/fxa/blob/45cbc1c5a2e716b54b467295835f161a578d9c08/.github/dependabot.yml#L7

Per https://github.blog/changelog/2021-05-21-dependabot-version-updates-can-now-ignore-major-minor-patch-releases/, it looks like we can ignore patch versions to avoid small PRs like _"chore(deps-dev): bump **xyz** from 7.18.9 to 7.18.10"_ and only focus on minor+major version bumps.

### Steps to reproduce

Clear steps describing how to reproduce the issue. Providing a URL, screenshot.

### Expected result

What should happen?

### Actual result

Here's the latest n [merged dependency PRs](https://github.com/mozilla/fxa/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Amerged+label%3Adependencies):

| TITLE | major.minor.patch
|:----|:----:|
chore(deps): bump aws-sdk from 2.1184.0 to 2.1189.0 | minor
chore(deps-dev): bump @babel/cli from 7.18.9 to 7.18.10 | **patch**
chore(deps-dev): bump sass from 1.52.1 to 1.54.3 | minor
chore(deps-dev): bump @storybook/addon-docs from 6.5.9 to 6.5.10 | **patch**
chore(deps): bump @emotion/react from 11.9.3 to 11.10.0 | minor
chore(deps-dev): bump ts-jest from 28.0.6 to 28.0.7 | **patch**
chore(deps): bump hapi-swagger from 14.5.1 to 14.5.2 | **patch**
chore(deps-dev): bump @storybook/html from 6.4.13 to 6.5.9 | minor
chore(deps-dev): bump @playwright/test from 1.23.4 to 1.24.0 | minor
chore(deps-dev): bump got from 11.8.3 to 11.8.5 | **patch**
chore(deps): bump aws-sdk from 2.1135.0 to 2.1184.0 | minor
chore(deps): bump slackapi/slack-github-action from 1.19.0 to 1.21.0 | minor

TLDR: Out of the last 12 PRs; 7 were minor (58.3%), 5 were patch (41.6%).

### Environment

What browser and operating system are you using?

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-5684)

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

I like that idea.  Let’s try it.

**Comment by cknowles-admin** [comment]
FxA is no longer synchronizing all issues between Jira and Github.  We are closing open issues and will selectively synchronize in the future.
